tests/performance_tests/README.md

```diff
@@ -0,0 +1,605 @@
+
+## Prerequisites
+
+Please refer to [Leap: Software Installation](https://github.com/AntelopeIO/leap#software-installation)
+
+## Steps
+
+1. Install Leap. For complete instructions on obtaining compiled binaries or building from source please refer to [Leap: Building From Source](https://github.com/AntelopeIO/leap#building-from-source)
+2. Run Tests
+    1. Full Performance Test Run:
+        ``` bash
+        ./build/tests/performance_tests/performance_test.py
+        ```
+    2. Single Performance Test Basic Run:
+        ```bash
+        ./build/tests/performance_tests/performance_test_basic.py
+        ```
+3. Collect Results - If specifying `--keep-logs` and/or `--save-json` and/or `--save-test-json`
+    1. Navigate to performance test logs directory
+        ```bash
+        cd ./build/performance_test/
+        ```
+    2. Log Directory Structure is hierarchical with each run of the `performance_test.py` reporting into a timestamped directory where it includes the full performance report as well as a directory containing output from each test type run (here, `performance_test_basic.py`) and each individual test run outputs into a timestamped directory that may contain block data loggs and transaction generator logs as well as the test's basic report.  An example directory structure could look like:
+        ``` bash
+        performance_test/
+        └── 2022-10-19_10-23-10
+            ├── report.json
+            └── testRunLogs
+                └── performance_test_basic
+                    ├── 2022-10-19_10-23-10
+                    │   ├── blockDataLogs
+                    │   │   ├── blockData.txt
+                    │   │   └── blockTrxData.txt
+                    │   ├── data.json
+                    │   └── trxGenLogs
+                    │       └── trx_data_output_7612.txt
+                    └── 2022-10-19_10-29-07
+                        ├── blockDataLogs
+                        │   ├── blockData.txt
+                        │   └── blockTrxData.txt
+                        ├── data.json
+                        └── trxGenLogs
+                            ├── trx_data_output_10744.txt
+                            └── trx_data_output_10745.txt
+        ```
+
+## Configuring Performance Tests
+
+### Performance Test
+
+`performance_test.py` can be configured using the following command line arguments:
+
+<details open>
+    <summary>Expand Argument List</summary>
+
+* `-p P`                  producing nodes count (default: 1)
+* `-n N`                  total nodes (default: 0)
+* `-d D`                  delay between nodes startup (default: 1)
+* `--nodes-file NODES_FILE`
+                    File containing nodes info in JSON format. (default: None)
+* `-s {mesh}`             topology (default: mesh)
+* `--dump-error-details`  Upon error print etc/eosio/node_*/config.ini and var/lib/node_*/stderr.log to stdout (default:
```

Comment:
Looks like Markdown is interpreting the underscores.  It's probably reasonable to put all of the command line help dumps into code format blocks.

Reply:
Addressed: [b641304](https://github.com/AntelopeIO/leap/pull/352/commits/b641304b5c62607aaac45e49a9d1de9cf701bba6)

Chose to stick with formatting as I had previously instead of full code block. I wrapped the path strings in a code block though to escape markdown interpreting * for italics.